example/example/spiders/books_05_1.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+"""
+Scrapy spider which uses Page Objects both for crawling and extraction.
+You can mix various page types freely.
```

Comment:
Could you please mention that this example shows how to skip scrapy Response?

Reply:
Done.